pt/deploy/README.md

```diff
@@ -59,6 +59,7 @@ Git irá controlar as alterações para todos os arquivos e pastas neste diretó
     __pycache__
     myvenv
     db.sqlite3
+    static/
```

Comment:
@scorphus Yep. As it is autogenerated on the PythonAnywhere step, there was some users sending the static files accidentally while using git on the same time they where trying to send the css. What do you think?